sitebridge-aws-dev-terraform/cluster/cluster.tf

```
@@ -0,0 +1,167 @@
+########################################################################
+# Variables Definitions
+########################################################################
+variable "vpc_id" {
+  type        = "string"
+  description = "VPC ID"
+}
+
+variable "subnet_id" {
+  type        = "string"
+  description = "subnet id to create the instance in"
+}
+
+variable "resourceName" {
+  type        = "string"
+  description = "Name to be used on a resource as cluster name as prefix"
+}
+
+variable "numInstances" {}
+
+variable "natpool_cidr_block" {
+  type = "string"
+}
+
+variable "igw_id" {
+  type = "string"
+}
+
+variable "vpc_security_group_ids" {
+  description = "A list of security group IDs to associate with"
+  type        = "list"
+}
+
+variable "kingdom" {
+  default = ""
+}
+
+variable "platformName" {
+  default = ""
+}
+
+variable "sitebridge" {}
+
+variable "sitebridgeImage" {}
+
+variable "amiList" {
+  type = "list"
+}
+
+variable "externalIPs" {
+  type = "list"
+}
+
+variable "instanceNamePrefix" {}
+
+variable "instanceNameSuffix" {}
+
+variable "instance_type" {
+  default = "c4.xlarge"
```

c5.4xlarge needed for something with the ipsec config.
```suggestion
  default = "c4.xlarge"
```
```suggestion
  default = "c4.xlarge"
```
```suggestion
  default = "c5.4xlarge"
```

---

Good catch